### ISSUE TITLE
Crash if obj path does not exist

### ISSUE THREAD

**Issue by nico87** [skiselkov/librain#7]
If the obj path filled in the librain/glass_n/obj dataref is wrong, the plugin crashes, at least on Linux.
Not a game stopper, but I thought it would be nice to know.

Thanks!

**Comment by kukel** [comment]
Do you have a log file with a crash? Maybe I can create a pull request then for this plugin to prevent it from crashing and just log an error

**Comment by nico87** [comment]
Not really. A lot of time has passed. It's actually pretty simple to re-create even tho the crash log is not simple to get.

**Comment by kukel** [comment]
Ok, then I'm not going to spend time on it...  unless you can recreate and have a log...
